runtime/libia2/init.c

```diff
@@ -1,7 +1,9 @@
 #include "ia2.h"
 #include "ia2_internal.h"
+// #include "ia2_allocator.h"
```

Comment:
How am I supposed to include `ia2_allocator`?  That's where `shared_realloc` seems to be defined.

Reply:
@fw-immunant what do you think about calling `__real_realloc` here instead of `shared_realloc`? I feel very uneasy about creating a dependency between the core libia2.a stuff and the partition alloc shim given our experience with firefox.